src/main/resources/format.yaml

```diff
@@ -4,6 +4,7 @@ statusUrl: "https://github.com/xstefank/test-repo/blob/master/.github/pull_reque
 
 format:
   title: "\\[WFLY-\\d+\\]\\s+.*|WFLY-\\d+\\s+.*"
+  commit: "\\[WFLY-\\d+\\]\\s+.*|WFLY-\\d+\\s+.*"
```

Comment:
One additional question - do you think it would be possible to avoid copy-paste of the regex if title and commit messages are expected to be same? Can you come up with some solution to this problem?